### PR TITLE
[FIX - Organizational Structure] Populating organizational structure tree view with api

### PR DESCRIPTION
**Pull request by fnormand01**
# Description

Reference: PR #422.

I needed to revert this PR because it was causing migrations errors. I fixed this issue by creating a new migration instead of updating an old one that has already been migrated.

### CODE REVIEW

File: frontend/src/text_resources.js [hunk 1]
```diff
@@ -208,11 +208,26 @@ let LOCALIZE = new LocalizedStrings({
...
       //Home Page
       homePage: {
+        testTitle: "The Sample eMIB Test",
         welcomeMsg: "Welcome to the eMIB Sample Test"
       },
 
       //HowTo Page
       howToPage: {
+        introductionPage: {
+          title: "Overview",
```
Comment (MichaelCherry) [inline frontend/src/text_resources.js]: Could you double check: I don't think these are used anywhere anymore...

File: frontend/src/text_resources.js [hunk 2]
```diff
@@ -786,11 +789,26 @@ let LOCALIZE = new LocalizedStrings({
...
       //Home Page
       homePage: {
+        testTitle: "La test pratique de BRG-e",
         welcomeMsg: "Bienvenue dans le test pratique de BRG-e"
       },
 
       //HowTo Page
       howToPage: {
+        introductionPage: {
+          title: "Aperçu général",
```
Comment (MichaelCherry) [inline frontend/src/text_resources.js]: see previous comment

**Comment by caleybrock** [comment]
@MichaelCherry can you review this? I think you had a better understanding of what happened the first time.

File: frontend/src/text_resources.js (hunk 1 above)
Comment (fnormand01) [reply frontend/src/text_resources.js]: Removed in commit #https://github.com/code-for-canada/project-thundercat/pull/435/commits/1f893d658222afa311f49821494f9f6bc659ffae

File: frontend/src/text_resources.js (hunk 2 above)
Comment (fnormand01) [reply frontend/src/text_resources.js]: Removed in commit #https://github.com/code-for-canada/project-thundercat/pull/435/commits/1f893d658222afa311f49821494f9f6bc659ffae

**Pull request merged**